app_legacy/Platform/Actions/UpdateGameWeightedPoints.php

```diff
@@ -10,11 +10,11 @@ class UpdateGameWeightedPoints
 {
     public function run(int $gameId): bool
     {
-        $query = "SELECT ach.ID, ach.Points, SUM(aw.HardcoreMode) AS NumAchieved
+        $query = "SELECT ach.ID, ach.Points, SUM(CASE WHEN ua.Untracked IS NULL OR NOT ua.Untracked THEN aw.HardcoreMode ELSE 0 END) AS NumAchieved
```

Comment:
Untracked should [not be nullable](https://github.com/RetroAchievements/RAWeb/blob/47f3e56f4933b89f5f22e990d9e0e21e960339a7/database/migrations/legacy/2012_10_03_133633_create_base_tables.php#L714).
```
show create table UserAccounts
```
```
  `Untracked` tinyint(1) NOT NULL DEFAULT '0',
```

Reply:
Resolved by https://github.com/RetroAchievements/RAWeb/pull/1484/commits/6297ca3920cb5a7261cf249a0fbe9d824447171b